auth-api/flags.json

```diff
@@ -0,0 +1,7 @@
+{
+    "flagValues": {
```

Comment:
is this needed?

Reply:
I think in LEAR It's mainly used for loading in some dummy flags from a local file when we're initializing the flag service without an SDK key while app.env isn't production.

I thought it might be useful in auth too so I pulled it over, Although it's not doing anything right now.

Reply:
It can be used for testing purposes.
If you want to run unit or other tests, and as a dev you don't have access to change the LD environment, you can use the local file to exercise the flag values.